fe/src/main/java/org/apache/doris/transaction/GlobalTransactionMgr.java

```diff
@@ -740,6 +683,13 @@ public void finishTransaction(long transactionId, Set<Long> errorReplicaIds) thr
                 for (PartitionCommitInfo partitionCommitInfo : tableCommitInfo.getIdToPartitionCommitInfo().values()) {
                     long partitionId = partitionCommitInfo.getPartitionId();
                     Partition partition = table.getPartition(partitionId);
+                    if (partition.getVisibleVersion() != partitionCommitInfo.getVersion() - 1) {
+                        LOG.debug("transactionId {}  VisibleVersion {}, CommitIn version {}. need wait",
+                                transactionId,
+                                partition.getVisibleVersion(),
+                                partitionCommitInfo.getVersion());
+                        return;
+                    }
```

Comment:
Remove this judgment block after the following `if (partition == null)` block.

Reply:
done